### ISSUE TITLE
Is this project still developed or maintained?

### ISSUE THREAD

**Issue by verdverm** [olekukonko/tablewriter#213]

**Comment by craigbox** [comment]
I'd be interested in contributing to maintaining a fork if the original authors/maintainers would bless such a thing. @olekukonko @mattn

**Comment by olekukonko** [comment]
@craigbox Yes the project is active, and contributors are welcome, what do you have in mind ?

**Comment by craigbox** [comment]
I am a maintainer of an open source tool that uses tablewriter, and I'd love to see fixes merged upstream so everyone can use them. I understand that your circumstances may not enable you to participate yourself as much as you once did, but please know there are people out there who are willing to help.

There are a number of fixes proposed in pull requests ([here is the one I care the most for](https://github.com/olekukonko/tablewriter/pull/207)) and a number of people are maintaining improvements in their own forks ([here is one I know of personally](https://github.com/fredbi/tablewriter)).

The last commit was over a year ago, and people are creating issued asking "is the project still maintained" 🙂

I don't consider myself capable of doing code review, though I could probably find some time from people on my team to help clear the backlog.  I personally could help with setting up standards and [triaging](https://github.com/olekukonko/tablewriter/pull/184) [obvious](https://github.com/olekukonko/tablewriter/pull/145) [things](https://github.com/olekukonko/tablewriter/pull/210/files).

We could also put tablewriter up for an [LFX mentorship](https://github.com/cncf/mentoring/tree/main/lfx-mentorship) for the next term, where we get a student contributor help address outstanding issues.

**Comment by olekukonko** [comment]
@craigbox thank you so much for the response. I have been busy hence why some of the issues were left unattended and I truly apologies 

Contribution, Support & Donations are very much welcome. I have created a new branch where I will start fixing most of the issues while ensuring the code is backwards compatible.

**Comment by craigbox** [comment]
How are things going, @olekukonko?  I saw some work on the prototype branch but no new tag yet.

**Comment by olekukonko** [comment]
@craigbox it's been a while ... you may want to look at the prototype branch to see what have been able to come up with ....  you can use https://github.com/olekukonko/tablewriter/issues/237 to track the progress

**Comment by olekukonko** [comment]
this is fully maintained and will be pushing the latest to master
